function-support.html

```diff
@@ -2692,13 +2692,13 @@ <h2 id="symbols">Symbols and Punctuation</h2>
 			<td>\(\triangleright\)</td>

```

Comment:
I think `--` and `---` need to be wrapped in `\text` too.  (In math mode, you get multiple minus signs.)